crates/type-system/Cargo.toml

```diff
@@ -0,0 +1,40 @@
+[package]
+name = "type-system"
+version = "0.0.0"
+edition = "2021"
+authors = ["HASH"]
+publish = false
+description = "Definitions of types within the Block Protocol Type System"
+
+[lib]
+name = "type_system"
+crate-type = ["cdylib", "rlib"]
+
+[features]
+default = ["console_error_panic_hook"]
+
+[dependencies]
+serde = { version = "1.0.137", features = ["derive"] }
+thiserror = "1.0.32"
+url = "2.2.2"
+wasm-bindgen = { version = "0.2.63", features = ["serde-serialize"] }
```

Comment:
I think it's possible to only pull it for the wasm target:

```toml
[target.'cfg(target_arch = "wasm32")'.dependencies]
wasm-bindgen = { version = "0.2.63", features = ["serde-serialize"] }
```

Reply:
I've moved most of the wasm stuff under that heading